### DISCUSSION TITLE
Proposal: Module backwards compatibility

### DISCUSSION THREAD

**Discussion by mpvl** [cue-lang/cue#2451] [Proposal]
Proposal https://github.com/cue-lang/cue/discussions/2330 relies on MVS, which requires a notion of backwards compatibility. This is discussed in this sub-proposal:
https://github.com/cue-lang/proposal/blob/main/designs/modules/2451-modules-compat.md

**Comment by myitcv** [comment]
This proposal was superseded by #2939. As we updated in https://github.com/cue-lang/cue/discussions/2939#discussioncomment-9468945, that proposal (and its sub-proposals) have now been accepted, hence we are closing this discussion to represent the decision on the v3 proposal.
